Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -589,9 +605,7 @@ class MessageDetailViewController: OWSViewController, MediaGalleryDataSourceDele
             return
         }
 
-        messageBubbleView.contentWidth = contentWidth()
-
-        let messageBubbleSize = messageBubbleView.size(forContentWidth: contentWidth())
+        let messageBubbleSize = messageBubbleView.measureSize()
```

Comment:
FWIW the old interface where messageBubbleView doesn't maintain state about it's layout context made more sense to me.  It seems like the VC should be directing the views, not the other way around.